.buildkite/pipeline.yml.template

```diff
@@ -0,0 +1,50 @@
+steps:
+  - label: ':nodejs: Run JavaScript checks'
+    plugins:
+      - docker#v3.5.0:
+          image: "node:12"
```

Comment:
Who controls this image? Can we use the nodethon container instead? I'd like to control our containers so we don't have to chase build breaks because some other project released a buggy container.

Reply:
It does nod have namespace, it's official docker image of nodejs. I dont like the idea of using nodethon, because python is not needed for pure javascript part. And nodethon uses node v13, that is not LTS version. Also, nodethon does not have yarn inside.

Reply:
What we decide here?

Reply:
As discussed with Scott, we can move forward with a greater autonomy so if you think we should use the official image then that's that.